src/components/Content.tsx

```diff
@@ -44,6 +44,7 @@ const Content: FC = () => {
     return children.length
   })
   const isAbsoluteContext = useSelector(state => isAbsolute(state.rootContext))
+  const longPress = useSelector(state => state.longPress)
```

Comment:
This will cause `Content` and thus the entire `LayoutTree` to unnecessarily re-render when `state.longPress` changes.

`useSelector` when the value needs to be reactive. If you need to access the Redux state in response to a user event, I prefer dispatching a thunk which gives you `getState()`.